### DISCUSSION TITLE
The billing information is "wrong"

### DISCUSSION THREAD

**Discussion by igoreq25** [community/community#103028] [GitHub Education] [answered]
Hello everyone,

I've been trying to apply for the Student Developer Pack since about a week now. At first, I believe I got reviewed manually, since it took a few days, but now I just keep getting rejected right after I click on "Process my application", regardless of what I upload.

Each time I'm told that my billing information is wrong, but I checked it many times (and also tried multiple variations of it), but nothing seams to help it. This is the message I get:

> Unfortunately, we weren't able to approve your educational discount request for [@igoreq25](https://github.com/igoreq25).
> 
> 
> What happened?
> The item you uploaded is insufficient to demonstrate your current academic status.
> Please see suggestions.
> Suggested proof to expedite your approval:
> 
> Please correct your [GitHub billing information](https://github.com/settings/billing/payment_information) with your last name exactly as it appears in your academic affiliation document. If you have only a single legal name, enter it in both the first and last name fields.

This happens regardless of weather I use my student ID or my year certificate.

Sometimes the system rejects me, without even giving me suggestions.

Attempting to contact GitHub about the issue directly has been just as frustrating - their support page won't even let me open a ticket about this, hence "[they] no longer provide individual manual reviews"...

Does anyone know how I could get verified in this situation? I'll be grateful for any tips

**Comment by igoreq25** [comment]
Update: I have made yet another attempt and it did not work.

**Comment by JCCourt** [comment]
It might be your billing last name is different to your student id last name or something.

**Reply by igoreq25** [reply]
That’s so thoughtful of you, thanks for the feedback!
Sadly after 29 unsuccessful attempts with several name and last name variations, I’m getting the idea it might not be it after all…

**Comment by mishmanners** [comment]
For this, you'll need to open a support ticket.

Please reach out to [GitHub Support](https://support.github.com/) and open a ticket. They should be able to help you.

**Reply by igoreq25** [reply]
Hello there, I've already tried that. As soon as you select that you found a bug or got rejected for the Student Developer Pack, a pop-up appears telling you that "[they] no longer provide individual manual reviews" and that therefore a ticket **cannot be created**.

For clarification, here is a screenshot:
![Screenshot](https://github.com/community/community/assets/60524567/e791f9b9-ffbf-458f-8792-852d185d1e27)

**Comment by centhoang** [comment]
I am having the same problem, after many different attempts, I still got this "name incorrect" message

**Comment by pozitp** [comment]
I have same problem too, my ID is not on English language and every time I got error about wrong first and second name in billing. Without manual verification, I don't know how to pass this. In the past I renewed my student pack 2 times, but now I can't.

**Reply by BlackRandomCraft** [reply]
same thing, can't confirm, the original is in Russian(((

**Comment by centhoang** [comment]
I know it now guys, the name in billing information must match the name in your Student ID Card in which photo you are taking
This is the problem in my case

Finally, I did it

**Reply by igoreq25** [reply]
Hi there, I'm glad it worked for you!
In my case it does not accept it regardles of what I type in... and I've tried quite a few ways of writing it.
Perhaps it is because my document is handwritten, and the application form cannot recognize it correctly :/

**Comment by samus-aran** [comment, marked as answer]
Hi @igoreq25 👋 

Although the verification is pretty robust it may at times need a little more assistance when it comes to particular languages and fonts (especially handwriting).

Here is a few quick tips on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile/billing information. I would suggest this also if you have difficult to read handwriting.. try re-writing important info in uppercase and position it next to your evidence. If you are unable to read its text on the image itself then the verification process may not be able to either.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- We request students submit a live photo of their evidence rather than an upload (in special circumstances).

Let us know how you get on 👍

**Reply by igoreq25** [reply]
Hello @samus-aran !
Thank you for reaching out;
I have printed out my name and submitted a live photo of it next to my school ID. I can see on the application page that the status is 'Approved'. However, I have not yet received an email confirmation or my benefits.

**Reply by samus-aran** [reply]
Yay 🎉 

The first stage is being Approved.  You'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within **72 hours of this approve status**. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

If you don't receive your academic benefits by the 5th day feel free to re-open this discussion 👍

**Comment by carlosferreyra** [comment]
May 23, 2024 
I uploaded a picture of my Academic Enrollment Book (Libreta Universitaria) and  ont top of it, a paper with ***UPPER CASE HANDWRITING*** transription of the ***_handwritten_*** parts of the document. ✅ and also changed my **username** to contain my _name_ and _last name_.

the original document is in Spanish so the OCR can recognize the alphabet
